pkg/sdn/plugin/node.go

```diff
@@ -204,9 +202,23 @@ func (node *OsdnNode) Start() error {
 		}
 	}
 
-	// Wait for kubelet to init the plugin so we get a knetwork.Host
-	log.V(5).Infof("Waiting for kubelet network plugin initialization")
-	<-node.kubeletInitReady
+	log.V(5).Infof("Waiting for kubelet and network plugin initialization")
+	err = kwait.ExponentialBackoff(
```

Comment:
also, I'm not sure this is the right timeout... shouldn't we poll indefinitely, until systemd kills us for failure to start?